setup.py

```
@@ -1032,6 +1033,21 @@ def make_relative_rpath_args(path):
             return ["-Wl,-rpath,@loader_path/" + path]
         elif IS_WINDOWS:
             return []
+        elif platform.system() == 'AIX':
+            try:
+                aix_ld_flags = os.environ['LDFLAGS']
+                install_rpath = ""
+                if "blibpath" in aix_ld_flags:
+                    start_pos = aix_ld_flags.find('-blibpath:')
+                    start_pos += len('-blibpath:')
+                    end_pos = aix_ld_flags.find(' ', start_pos)
+                    if end_pos == -1:
+                        install_rpath = aix_ld_flags[start_pos:bpath:]
+                    else:
+                        install_rpath = aix_ld_flags[start_pos:end_pos]
+                return ["-Wl,-blibpath:" + install_rpath + path]
+            except:
```

a bit dangerous here because it silently falls back and that could be due to a parse error, a KeyError, etc and it doesn't propagate any of that info to the user.

---

Hmm. How should we handle this then? Should we  display a message to the user?

---

@Skylion007 Kindly check the logic now.